core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/MethodConfigurationComponent.scala

```diff
@@ -167,8 +171,9 @@ trait MethodConfigurationComponent {
         }
     }
 
-    def hideMethodConfigurationAction(id: Long, name: String): ReadWriteAction[Int] = {
-      findById(id).map(rec => (rec.deleted, rec.name)).update(true, name + "_" + DateTime.now().toString("yyyy-MM-dd_HH:mm:ss"))
+    def hideMethodConfigurationAction(id: Long, methodConfigName: String): ReadWriteAction[Int] = {
+      findById(id).map(rec => (rec.deleted, rec.name))
+        .update(true, methodConfigName + "_" + DateTime.now().toString("yyyy-MM-dd_HH:mm:ss"))
```

Comment:
After much in-person conversation, yes the timestamps are needed. The submission record references method config by database ID, but the case class uses the MC name, so trashing the timestamp makes us lose information (and potentially repoint submissions to the wrong method configurations on resave).